spec/features/client_slug_authz_spec.rb

```diff
@@ -69,9 +69,7 @@
     proposal_path = current_path
     visit proposal_path
     expect(page.status_code).to eq(200)
-    add_as_observer(gsa_user)
-    expect(page.status_code).to eq(403)
-    expect(page).to have_content("You are not allowed to add observers to that proposal.")
+    expect_to_not_select('observation_user_email_address', gsa_user.email_address)
```

Comment:
perhaps `expect_to_not_see_select_option` ? when I first saw this, I thought it was for whether someone was selecting something or not, which was a big confusing given that it's an expectation.